casbin/core_enforcer.py

```diff
@@ -440,9 +440,7 @@ def enforce_ex(self, *rvals):
         req_str = req_str + ", ".join([str(v) for v in rvals])
 
         req_str = req_str + " ---> %s" % result
-        if result:
-            self.logger.info(req_str)
-        else:
+        if not result:
```

Comment:
I disagree, I think the log should be logged regardless of `true` or `false`.

Reply:
@leeqvip is there a way for user to turn off the log prints?

Reply:
There should be a way to turn off the logs, this is unnecessary printing the logs and which impacts the server start time. Most of the people are facing this issue currently.

Reply:
Plz follow the Go Casbin's way to make logging configurable.

@Nekotoxin can you do this?

Reply:
Control whether to print the log by setting the log level.

Reply:
@anandthati can you follow the way to change it?